### PR TITLE
Api keys

### PR DESCRIPTION
**Pull request by symroe**
1. Rate limit unauthenticated users
2. Give everyone an API key
3. Make an MVP profile page with the API key on it (and some other stuff)
4. Document how to authenticate
5. Update the `live_site_importer` to rate limit itself and use an API key if we have one

### CODE REVIEW

File: ynr/apps/wombles/templates/wombles/my_profile.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>My profile</h1>
+    <p>Your username is {{ user.username }} and your contact email is {{ user.email }}.</p>
+    <p>When you craete an account we create an API key for you, this lets you use <a href="{% url "api-home" %}">our API</a>.</p>
```
Comment (VirginiaDooley) [inline ynr/apps/wombles/templates/wombles/my_profile.html:6]: typo "create"

File: ynr/apps/api/templates/api/api-home.html [hunk 2]
```diff
@@ -13,6 +13,20 @@
 * Please give credit to Democracy Club when reusing this data.
 * Please [read the full terms]({% url "api-terms" %}).
 
+### Authentication
+
+The API has a rate limit of 10 requests per minute for unauthenticated users.
+{% if request.user.is_authenticated %}
+To authenticate, grab the token from <a href="{% url "my_profile" %}">your profile page</a>.
+
+Pass this token in with `auth_token={{ request.user.auth_token.key }}`.
```
Comment (VirginiaDooley) [inline ynr/apps/api/templates/api/api-home.html:22]: Do we want to render the token here if we are telling them to grab it from the profile page? Not a big deal but maybe we do one or the other.

File: ynr/apps/wombles/templates/wombles/my_profile.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>My profile</h1>
+    <p>Your username is {{ user.username }} and your contact email is {{ user.email }}.</p>
+    <p>When you create an account we create an API key for you, this lets you use <a href="{% url "api-home" %}">our API</a>.</p>
+    <p>Your API key is <code>{{ user.auth_token.key }}</code>. You can use this by passing <code>?auth_token={{ user.auth_token.key }}</code> to API requests.x</p>
```
Comment (GeoWill) [inline ynr/apps/wombles/templates/wombles/my_profile.html:7]: Is it worth also mentioning the header method/giving a curl example - something like: 
```
curl -X GET candidates.democracyclub.org.uk/api/example/ -H 'Authorization: Token {{ user.auth_token.key }}'
```

**Review by GeoWill — APPROVED** [review]

**Comment by chris48s** [comment]
Drive by: Probably not the most critical one, but one of the things that will hit the rate limits is Orinoco. You'll need to either give it an API key, or (easier) bump up the sleep in https://github.com/DemocracyClub/womblr/blob/4cfb6da6bce36bc61994a40ac97e8bcdf39d4eab/scraper.py#L222 so it waits longer.

I wonder what else is going to excitingly fall over? :)

**Comment by symroe** [comment]
Good point...we're actually going to give womblr some love soon, but I'll add an API key anyway

**Pull request merged**